packages/spark-core/components/_accordions.scss

```diff
@@ -32,9 +32,7 @@
   text-decoration: none;
 
   &:hover,
-  &:active,
-  &:focus,
-  &:visited {
+  &:active {
```

Comment:
should probably have some sort of focus state on this. Also, does removing visited instead of styling it make the default visited state bleed through? I don't know what that would actual look like.

Reply:
it still has the blue outline for focus, just not the blue background / white color anymore. 
I'll test it across browsers for the visited color coming through.

Reply:
Tested across browsers, no bleeding. I think its because the color of the text in the link is actually controlled by the h3 thats surrounding it. The bug was there because we were applying a color change to the h3's class when the outer link was :visited